external/js/cothority/src/byzcoin/proto/requests.ts

```diff
@@ -224,6 +224,55 @@ export class GetSignerCountersResponse extends Message<GetSignerCountersResponse
     }
 }
 
+/**
+ * Holds an instanceID and it's latest known version.
+ */
+export class IDVersion extends Message<IDVersion> {
+    static register() {
+        registerMessage("IDVersion", IDVersion);
+    }
+
+    readonly id: Buffer;
+    readonly version: Long;
+}
+
+/**
+ * Requests a set of proofs that will be returned in a single message.
+ * The reply will only hold proofs for instances that have a higher version than the ones given here.
+ */
+export class GetUpdatesRequest extends Message<GetUpdatesRequest> {
+    static readonly sendVersion0 = Long.fromNumber(1);
+
+    static register() {
+        registerMessage("GetUpdatesRequest", GetUpdatesRequest);
+    }
+
+    readonly instances: IDVersion[];
+    readonly flags: Long;
+    readonly latestblockid: Buffer;
+
+    constructor(props?: Properties<GetUpdatesRequest>) {
```

Comment:
Useless, only forward args to superclass.

Reply:
copy/pasted from other definitions...